Puzzlepart/prosjektportalen365
# Script to fix/update aggregated webpart pages

### Your checklist for this pull request


### Description

Adds a new script that overwrites the aggregated ClientSidePages with the new changes to fix issues users are having.

### How to test

1. Run script 
2. Check to see that aggregated ClientSidePages are working


### Relevant issues (if applicable)

#754
